docs/source/guides/build.rst

```diff
@@ -165,64 +153,57 @@ Install CMake, Git, Microsoft compilers via chocolatey being in Administrative m
     # Don't forget the password you set!
 
 
-Vcpkg Dependency Manager
-""""""""""""""""""""""""
 
-Although Vcpkg is aimed to control dependency hell among the C++ libraries,
-unfortunately, we cannot install its default version.
-The first problem is that Iroha dependency called SOCI is not able to work with the latest Boost.
-The second reason - Vcpkg does not provide Postgres related libraries for Debug build.
+Install build dependencies with Vcpkg Dependency Manager
+--------------------------------------------------------
 
-The solution is to use Vcpkg from a `pull request <https://github.com/Microsoft/vcpkg/pull/6328>`_ (until it is merged):
 
-.. code-block:: shell
+Currently we use Vcpkg as a dependency manager for all platforms - Linux, Windows and MacOS. We use a fixed version of
+Vcpkg to ensure the patches we need will work. The version can be found inside the Iroha repository so we need to
+clone Iroha first.
+The whole process is pretty similar for all platforms but the exact commands are slightly different.
 
-  git clone https://github.com/Microsoft/vcpkg.git --depth=1
-  cd vcpkg
-  git fetch origin pull/6328/head:vcpkg_for_iroha
-  git checkout vcpkg_for_iroha
+Linux and MacOS
+^^^^^^^^^^^^^^^
 
-Then follow Vcpkg installation `guide <https://github.com/Microsoft/vcpkg/blob/master/README.md>`_:
+Run in terminal:
 
-.. code-block:: text
+.. code-block:: shell
 
-  # execute in Power shell
-  .\bootstrap-vcpkg.bat
-  .\vcpkg.exe integrate install
+  git clone https://github.com/hyperledger/iroha.git
+  git clone https://github.com/Microsoft/vcpkg.git
+  cd vcpkg
+  git checkout $(cat ../iroha/vcpkg/VCPKG_COMMIT_SHA)
+  for i in ../iroha/vcpkg/patches/*.patch; do git apply $i; done;
+  ./bootstrap-vcpkg.sh
+  ./vcpkg install $(cat ../iroha/vcpkg/VCPKG_DEPS_LIST | cut -d':' -f1 | tr '\n' ' ')
+  ./vcpkg install --head $(cat ../iroha/vcpkg/VCPKG_HEAD_DEPS_LIST | cut -d':' -f1 | tr '\n' ' ')
```

Comment:
I believe this script should be placed in a separate file like prepare_vcpkg.sh and used both in docker and by dockerless Linux users. Otherwise the code is duplicated in the first and there is too much copy-paste for the second.